programs/launcher/main.cpp

```diff
@@ -840,6 +866,38 @@ launcher_def::write_config_file (tn_node_def &node) {
   cfg.close();
 }
 
+void
+launcher_def::write_logging_config_file(tn_node_def &node) {
+  bf::path filename;
+  eosd_def &instance = *node.instance;
+
+  bf::path dd = stage/ instance.data_dir;
+  if (!bf::exists(dd)) {
+    bf::create_directory(dd);
+  }
+
+  filename = dd / "logging.json";
+
+  bf::ofstream cfg(filename);
+  if (!cfg.good()) {
+    cerr << "unable to open " << filename << " " << strerror(errno) << "\n";
+    exit (9);
+  }
+
+  auto log_config = fc::logging_config::default_config();
+  log_config.appenders.push_back(
+        fc::appender_config( "net", "gelf",
+            fc::mutable_variant_object()
+                ( "endpoint", "10.160.11.21:12201" )
```

Comment:
Hardcoded? This needs to be configurable.

Reply:
Next version.  There's only one gelf server today.

Reply:
Hardcoded value will be addressed by: STAT-268